expo/scheduler/templates/scheduler/allocate_volunteers.tmpl

```diff
@@ -35,8 +32,8 @@ included in event edit flow. Show WorkerAllocationForms
 	  {{field}}
 	{% endfor %}
 	  <td>  <input type = "submit" name="edit" value="Edit/Create Allocation"></td>
-	  {% if form.alloc_id.value > 0  %}
-	  <td>  <input type = "submit" name="delete" value="Delete Allocation"></td>
+	  {% if form.alloc_id.value > 0 and form.alloc_id.value != '-1' %}
```

Comment:
Shouldn't this be addressed upstream? If we're sometimes passing an int and sometimes we're passing the string "-1", then that's pretty bad. If we're not going to fix this in this PR, then this should be filed as an issue.